Source/ThirdParty/libwebrtc/Source/webrtc/modules/rtp_rtcp/source/rtp_format_h264.cc

```diff
@@ -239,6 +239,7 @@ bool RtpPacketizerH264::NextPacket(RtpPacketToSend* rtp_packet) {
     // Single NAL unit packet.
     size_t bytes_to_send = packet.source_fragment.size();
     uint8_t* buffer = rtp_packet->AllocatePayload(bytes_to_send);
+    RTC_DCHECK(buffer);
```

Comment:
I am not sure we need this one, given we memcpy, we will get a crash anyway the line after.

Reply:
I removed all the `RTC_DCHECK()` expressions added since they would crash with a `nullptr` anyway.